apps/dashboard/app/controllers/application_controller.rb

```diff
@@ -38,6 +38,10 @@ def usr_apps
     @usr_apps ||= ::Configuration.app_sharing_enabled? ? UsrRouter.all_apps(owners: UsrRouter.owners) : []
   end
 
+  def all_apps
+    @all_apps ||= sys_apps + usr_apps + dev_apps
```

Comment:
Instead perhaps this should be `nav_all_apps`? 

    @nav_all_apps ||= nav_sys_apps + nav_usr_apps + nav_dev_apps

Reply:
Something like `frame-renderer` at OSC doesn't appear in the nav.  There's no NavConfig.category for 'Gateways Apps'

Reply:
I see. Done.